pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BaseBrokerStarter.java

```diff
@@ -85,6 +85,8 @@
   protected String _hostname;
   protected int _port;
   protected String _instanceId;
+  private boolean _isStarting = false;
```

Comment:
They might need to be `volatile` as they can be accessed from a different thread